cirq-core/cirq/sim/act_on_args.py

```diff
@@ -213,22 +212,6 @@ def __iter__(self) -> Iterator[Optional['cirq.Qid']]:
     def _act_on_fallback_(self, action: Any, qubits: Sequence['cirq.Qid'], allow_decompose: bool):
         """Handles the act_on protocol fallback implementation."""
 
-    @property  # type: ignore
```

Comment:
I think we should remove it from the constructor here too since it's no longer used anywhere.